test

```diff
@@ -0,0 +1 @@
+web3 = eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweDNlNjkxM2Y1NDYzMDRjODkxNkY4QTE0M0I2QWMxODQ5RDAzMzE2NjIiLCJpc3MiOiJ3ZWIzLXN0b3JhZ2UiLCJpYXQiOjE2OTUxMzM3NTYwNDcsIm5hbWUiOiJkZiJ9.IMHt-KRffTRY92ryHSnFv6gY5MkB6O74MlzJd_Na9AI
```

Comment:
@ankushgoel27 this should be deleted! if the key is live it should be rotated too